packages/manager/modules/account-migration/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@ovh-ux/manager-account-migration",
+  "version": "0.0.1",
+  "private": true,
+  "description": "OVHcloud Migration product",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/modules/account-migration"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "main": "./src/index.js",
+  "devDependencies": {
+    "lodash": "^4.17.15"
+  },
+  "peerDependencies": {
+    "@ovh-ux/manager-core": "^9.0.0",
+    "@ovh-ux/ng-at-internet": "^5.0.1",
+    "@ovh-ux/ng-ovh-swimming-poll": "^5.0.1",
+    "@ovh-ux/ng-translate-async-loader": "^2.1.0",
+    "@uirouter/angularjs": "^1.0.20",
+    "angular": "^1.7.5",
+    "angular-translate": "^2.11.0",
+    "angular-ui-bootstrap": "~1.3.3",
+    "moment": "^2.24.0",
+    "ovh-api-services": "^9.39.1",
+    "ovh-ui-angular": "^3.16.3",
+    "ovh-ui-kit": "^2.42.8"
```

Comment:
Why targeting `migration-v4` if we're still using UI Kit v2 ?
This should be changed to `@ovh-ux/ui-kit`

Reply:
Missed removing it. Now it is fixed. Thanks